common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +99,847 @@ innovation_taming_the_skies = {
+		}
 	}
 
 	unlock_maa = aerial_unit
 
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @archmage
+
+	potential = {
+		AND = {
+			OR = {
+				this.culture_group = culture_group:human_group
+				this = culture:blood_elf
+				this = culture:high_elf
+			}
+			NOT = {
+				this = culture:pirate_culture
+				this = culture:wastewander	
+			}
+		}
+	}
+
+	unlock_maa = archmage
+
+	flag = global_maa
+}
+
+
+innovation_blademaster = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @blademaster
+
+	potential = {
+		this = culture:burning_blade
+	}
+
+	unlock_maa = blademaster
+
+	flag = global_maa
+}
+
+innovation_buccaneer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @buccaneer
+
+	potential = {
+		OR = {
+			this = culture:pirate_culture
+			this = culture:blackwater
+		}
+	}
+
+	unlock_maa = buccaneer
+
+	flag = global_maa
+}
+
+innovation_brute = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @brute
+
+	potential = {
+		this.culture_group = culture_group:gnoll_group
+	}
+
+	unlock_maa = brute
+
+	flag = global_maa
+}
+
+innovation_candleholder = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @candleholder
+
+	potential = {
+		this.culture_group = culture_group:kobold_group
+	}
+
+	unlock_maa = candleholder
+
+	flag = global_maa
+}
+
+innovation_dervishes = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dervishes
+
+	potential = {
+		this = culture:wastewander
+	}
+
+	unlock_maa = dervishes
+
+	flag = global_maa
+}
+
+innovation_dire_troll = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @dire
+
+	potential = {
+		this.culture_group = culture_group:troll_group
+	}
+
+	unlock_maa = dire_troll
+
+	flag = global_maa
+}
+
+innovation_dragon = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @flying
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:dragon_group
+			this = culture:dragonmaw
+		}
+	}
+
+	unlock_maa = dragon
+
+	flag = global_maa
+}
+
+innovation_gladiator = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @gladiator
+
+	potential = {
+		this = culture:shattered_hand
+	}
+
+	unlock_maa = gladiator
+
+	flag = global_maa
+}
+
+innovation_troll_ranged = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @headhunter
+
+	potential = {
+		this.culture_group = culture_group:troll_group
+	}
+
+	unlock_maa = headhunter
+
+	flag = global_maa
+}
+
+innovation_hexer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @hexer
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:troll_group
+			this.culture_group = culture_group:harpy_group
+			this = culture:saurok
```

Comment:
I don't see what we could use instead of this, if we use a saurok or harpy it wont work for the other two.

Maybe:
![image](https://user-images.githubusercontent.com/74790101/134786664-9f4ca441-7c3e-456d-8398-0a7ac81f0f81.png)

Reply:
but its not very warcrafty, and I think the troll in the picture, though obviously a troll, has a mask on and is pretty covered, so idk

Reply:
maybe we can just split up this tech since it causes so many problems, but even if we removed trolls, the saurok images and harpy images will come into conflict.

Reply:
> maybe we can just split up this tech since it causes so many problems, but even if we removed trolls, the saurok images and harpy images will come into conflict.

I would create a few units to solve it.